### DISCUSSION TITLE
Initialise SequelizeStorage

### DISCUSSION THREAD

**Discussion by mittalyashu** [sequelize/umzug#362] [Q&A] [answered]
I tried other ways than mentioned in the docs, none of them worked.

```js
const path = require("path");
const { Umzug, SequelizeStorage } = require("umzug");

const database = require(".");
const context = database.getQueryInterface();

const unzug = new Umzug({
	migrations: {
		path: path.resolve(__dirname, "migrations"),
		pattern: /\.js$/
	},
	context,
	storage: new SequelizeStorage({ database })
});

module.exports = unzug;
```

The above follows the docs says, in return shows an error `SequelizeStorage is not a constructor`.

**Comment by mmkal** [comment, marked as answer]
@mittalyashu which version of umzug are you using?

**Reply by mittalyashu** [reply]
`v2.3.0`

**Reply by mmkal** [reply]
It looks like you're mixing up v3 beta docs with v2. See the notice at the top of the readme for guidance on using the stable version. Or, install with `npm install umzug@beta` and follow the instructions for using v3. There will be fewer breaking changes and better support that way, but you'll be using a prerelease package.

**Reply by mmkal** [reply]
Marking this as "answered", feel free to comment if you have problems with v3 or v2 though.
